README.md

```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling
+
+If the test environment is too good, and the metrics are too big to catch dips in performance after a PR you can test it with `CPU throttling` of 2 with the commands:

```

Comment:
@stanca-pop-lgp @adrian-cocoara-lgp 
Could you please review again for commits after your review? This explanation looks not for README but PR description. At least we need to provide some available options and what it affects. A kind of `Throttling option` section would be better to explain what users are able to do something for their purpose.

Reply:
Sure, we will find a better description for this